packages/discovery-provider/src/api/v1/tracks.py

```diff
@@ -791,7 +792,7 @@ def get(self, track_id):
 
 # Download
 
-download_parser = reqparse.RequestParser(argument_class=DescriptiveArgument)
+download_parser = current_user_parser.copy()
```

Comment:
i have found that this pattern can be a little misleading. imo wet is best here, but not a strong opinion

Reply:
Oh interesting. I found this helpful because it means all of the parsers inherit the same docs and field name for current user id and changing it in one place changes all of them. I'd like to leave it this way for now since it's the convention for most of our parsers in the API to inherit `user_id` through copying this parser.